ui/app/helpers/mountable-secret-engines.js

```diff
@@ -1,6 +1,13 @@
 import { helper as buildHelper } from '@ember/component/helper';
 
-const MOUNTABLE_SECRET_ENGINES = [
+export const KMIP = {
+  displayName: 'KMIP',
+  value: 'kmip',
+  type: 'kmip',
+  category: 'generic',
+};
+
+export const MOUNTABLE_SECRET_ENGINES = [
```

Comment:
pretty sure we don't need to export this const anymore since we're only using the `KMIP` const elsewhere